Islandora-CLAW/islandora_demo
# Untie openseadragon and pdfjs EVAs from repository item bundles: Issue-1063

**GitHub Issue**: /Islandora-CLAW/CLAW/issues/1063

# What does this Pull Request do?

Removes the views config that binds the openseadragon and pdfjs EVAs to the repository item bundle. This allows other content types to use them without modifying the islandora_demo configs which enables a clean upgrade path.

# What's new?

* Changes openseadragon and pdfjs EVA views to such that any bundle can use them.

# How should this be tested?

- Apply the PR
- Feature import islandora_demo
- Create a new content type
- View the display settings for any content type (e.g. Article) and you will see the Openseadragon and PDFjs EVAs are now available.

# Additional Notes:

Perhaps these should actually be moved to separate sub-modules so repositories that aren't using Openseadragon or PDFjs don't have these cluttering their display config pages. 

# Interested parties
@dannylamb & @Natkeeran
